test/jdl/jdl-importer.spec.js

```diff
@@ -1937,5 +1937,30 @@ paginate * with infinite-scroll
                 ]);
             });
         });
+        context('when choosing neo4j as database type', () => {
+            let importState;
+
+            before(() => {
+                const content = `entity Person {
+   name String
+}
+
+relationship OneToMany {
+   Person{friends} to Person
+}`;
```

Comment:
indentation 😇

Reply:
There's no issue :)

Reply:
:o

Reply:
Is it the string templating thing that's misleading? If so, I'll extract the JDL content to a file.